src/azure-cli/HISTORY.rst

```diff
@@ -1609,6 +1609,7 @@ Release History
 
 **SQL VM**
 
+* Added more arguments to `az sql vm show --expand` command 
```

Comment:
I think you need to add again - for the latest version released (Example 2.0.16) a 
**SQL VM **
* Added more arguments to `az sql vm show --expand` command 

Adding on the old history won't show up on the latest released CLI.